node/services/config.go

```diff
@@ -23,39 +23,44 @@ type Config struct {
 	TrustedPeers []string
 }
 
-// TODO(@Wondertan): We need to hardcode trustedHash hash and one bootstrap peer as trusted.
 func DefaultConfig() Config {
 	return Config{
 		TrustedHash:  "",
 		TrustedPeers: make([]string, 0),
 	}
 }
 
-func (cfg *Config) trustedPeers() ([]*peer.AddrInfo, error) {
+func (cfg *Config) trustedPeers(net params.Network) (infos []*peer.AddrInfo, err error) {
```

Comment:
Not a big fan of named returns btw: https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values

But no biggie and you can decide to leave this as is or not.

Reply:
@liamsi I am also not a big fan of those. This is the case where it's simple and obvious + avoids an additional and useless `var err error` line by putting it to the return params.